be/src/formats/parquet/stored_column_reader.cpp

```diff
@@ -400,7 +400,9 @@ Status OptionalStoredColumnReader::_lazy_skip_values(uint64_t begin) {
         level_t* def_levels = nullptr;
         size_t level_parsed = 0;
         size_t cur_to_skip = row_to_skip - row_skipped;
-        RETURN_IF_ERROR(_decode_levels(&cur_to_skip, &level_parsed, &def_levels));
+        // skip batch by batch to avoid big memory alloc
+        size_t batch_to_skip = std::min(cur_to_skip, (size_t)8192);
```

Comment:
Please define constant value for magic number

Reply:
done